### PR TITLE
Adding paging options for fetchGrantedAccess method

### PR DESCRIPTION
**Pull request by greg-szrama**
Addresses PRO-579.

### CODE REVIEW

File: documentation/tools/dataprotector/methods/fetchgrantedaccess.md [hunk 1]
```diff
@@ -55,6 +71,35 @@ Filter the granted access by an authorized user address.
...
+    authorizedApp: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
+    authorizedUser: '0x4a3B7c1eF78fA81bDf1aAa314D09a97578b0CA37',
+    <strong>page: 1,</strong>
+    pageSize:100
+})
+</code></pre>
+
+### pageSize (optional)
+
+Specifies the number of records in each page of the resultset. This is used in conjunction with the optional `page` parameter to constrain the size of the result. The default for this value is `20` but this only comes into play when specifying a `page` ID.
```
Comment (Le-Caignec) [inline documentation/tools/dataprotector/methods/fetchgrantedaccess.md:96]:
```suggestion

Specifies the number of records in each page of the result set. This is used in conjunction with the optional `page` parameter to constrain the size of the result. The default for this value is `20` but this only comes into play when specifying a `page` parameter.
```

File: documentation/tools/dataprotector/methods/fetchgrantedaccess.md [hunk 2]
```diff
@@ -55,6 +71,35 @@ Filter the granted access by an authorized user address.
...
+    authorizedApp: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
+<strong>    authorizedUser: '0x4a3B7c1eF78fA81bDf1aAa314D09a97578b0CA37',</strong>
+    page: 1,
+    pageSize:100
+})
+</code></pre>
+
+### page (optional)
+
+Specifies the results page to return. If using this field you may also specify a `pageSize`. The default for this is `0` but if using paging, the functionality begins counting from index `1`.
```
Comment (Le-Caignec) [inline documentation/tools/dataprotector/methods/fetchgrantedaccess.md:83]:
```suggestion
Specifies the results page to return. If using this field you may also specify a `pageSize`. The default for this is `0` but if using paging, the functionality begins counting from the value specify for this parameter.
```

Comment (Le-Caignec) [reply documentation/tools/dataprotector/methods/fetchgrantedaccess.md:83]: it's not always 1

**Review by Le-Caignec — CHANGES_REQUESTED** [review]

File: documentation/tools/dataprotector/methods/fetchgrantedaccess.md (hunk 2 above)
Comment (greg-szrama) [reply documentation/tools/dataprotector/methods/fetchgrantedaccess.md:83]: What I meant here is it's 1-indexed rather than 0-indexed. I'll try to rephrase this to be more clear.

**Review by Le-Caignec — APPROVED** [review]
